vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -158,36 +163,31 @@ def waterfall(data_frame: pd.DataFrame, x: str, y: str, measure: list[str]) -> g
         measure (list[str]): List specifying the type of each bar, can be "relative", "total", or "absolute".
 
     Returns:
-        go.Figure: A Plotly Figure object representing the Waterfall chart.
+        go.Figure: A Plotly Figure object of the Waterfall chart.
 
     For additional parameters and customization options, see the Plotly documentation:
     https://plotly.com/python/reference/waterfall/
 
     """
-    fig = go.Figure(
-        go.Waterfall(
-            x=data_frame[x],
-            y=data_frame[y],
-            measure=data_frame[measure],
-        )
+    return go.Figure(
+        data=go.Waterfall(x=data_frame[x], y=data_frame[y], measure=data_frame[measure]),
+        layout={"showlegend": False},
     )
-    fig.update_layout(showlegend=False)
-    return fig
 
 
 @capture("graph")
-def radar(data_frame, **kwargs) -> go.Figure:
-    """Creates a radar chart using Plotly's `line_polar`.
+def radar(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
+    """Creates a radar chart using Plotly's px.line_polar.
```

Comment:
Would maybe go over the one-liners of all charts. Reason: sometimes we say what the underlying framework is (Plotly, px, go) sometimes we just mentioned the chart type. Would unify this a bit